content/version/1/4/code-of-conduct.id.md

```diff
@@ -34,28 +27,28 @@ yang positif meliputi :
 
 Contoh perilaku yang tidak dapat diterima oleh partisipan meliputi :
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
-  advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
-  address, without explicit permission
+* Penggunaan bahasa atau citra seksual dan perhatian seksual yang tidak diinginkan.
+* Trolling, komentar menghina, dan serangan terhadap individual atau pandangan politik.
+* Pelecahan umum atau pribadi
+* Menerbitkan informasi pribadi orang lain, seperti alamat fisik atau elektronik,
+tanpa izin.
+* Perilaku lain yang dapat dianggap tidak sesuai dalam pengaturan profesional
 * Other conduct which could reasonably be considered inappropriate in a
   professional setting
 
-## Our Responsibilities
+## Responsibilitas Kami
```

Comment:
hi, I found this thread from a tweet. I think it's better to use 'tanggung jawab' instead of 'responsibilitas'.